interface-elements-for-desktop/articles/report-designer/report-designer-for-winforms/use-report-elements/use-basic-report-controls/picture-box.md

```diff
@@ -15,25 +15,52 @@ To add the **Picture Box** control to a report, drag the **Picture Box** item fr
 
 Specify one of the following properties to set an image:
 
-- **ImageSource**  
-    Use this property to save images along with a report definition.
-- **ImageUrl**  
-    Use this property to save only the path to the image.
+- **Image Source**  
+    Save the image to the report's definition.
+- **Image Url**  
+    Save only a path to the image.
 
 ![](../../../../../images/eurd-win-picture-box-image-property.png)
 
+## Assign an External Image
+
+Click the **Image Source** / **Image URL** property's ellipsis button and the **Open File** dialog is invoked.
+
+![](../../../../../images/eurd-win-ImageSource-OpenFileDialog.png)
+
+The selected image / its URL is saved to the report definition **.repx** file.
```

Comment:
Do you mean "The selected image and its URL are saved..."

Reply:
No, / means or

"Click the **Image Source** / **Image URL** property's ellipsis button...

The selected image / its URL is saved..."

Reply:
Then we can say "The selected image or its URL is saved..."